java/zoola-academy-homework.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
.iml files should be added to gitignore

Reply:
Add .iml files to gitignore.

Reply:
you still need to remove the file from the MR, cause it appears in the list of all changed files

Reply:
I should remove zoola-academy-homework.iml file, right?

Reply:
you need to remove it from git and the PR, you can keep it on your local machine